src/stores/rulesStore.js

```diff
@@ -30,7 +30,8 @@ export const useRulesStore = defineStore('rules', {
     parsedRulesList: [],
     rulesetFilesUploaded: false,
     requestMatched: false,
-    matchedRule: {},
+    matchedRuleString: '',
+    matchedRuleStringSet: false,
```

Comment:
Hmm, thoughts on just representing this with `undefined` as the value of `matchedRuleString`? It seems odd to have an entirely separate bool.